can_tools/scrapers/official/PA/pa_state.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+
+import us
+from can_tools.scrapers import CMU, DatasetBase
+from can_tools.scrapers.official.base import ArcGIS
+
+from typing import Any
+
+
+class PennsylvaniaCasesDeaths(ArcGIS, DatasetBase):
+    """
+    Fetch county level covid data from Pennsylvania's ARCGIS dashboard
+    """
+
+    ARCGIS_ID = "Nifc7wlHaBPig3Q3"
+    has_location = False
+    location_type = "county"
+    state_fips = int(us.states.lookup("Pennsylvania").fips)
+    source = "https://experience.arcgis.com/experience/ed2def13f9b045eda9f7d22dbc9b500e"
+    service: str = "COVID_PA_Counties"
+
+    def fetch(self) -> Any:
+        return self.get_all_jsons(self.service, 0, 1)
+
+    def pre_normalize(self, data) -> pd.DataFrame:
+        df = self.arcgis_jsons_to_df(data)
+
+        # Make columns names all-lowercase
+        df.columns = [x.lower() for x in list(df)]
+
+        crename = {
+            "cases": CMU(
+                category="cases", measurement="cumulative", unit="unique_people"
+            ),
+            "deaths": CMU(
+                category="deaths", measurement="cumulative", unit="unique_people"
+            ),
+            "probable": CMU(
+                category="cases_probable",
+                measurement="cumulative",
+                unit="unique_people",
+            ),
+            "negative": CMU(
+                category="pcr_tests_negative",
+                measurement="cumulative",
+                unit="unique_people",
+            ),
+            "confirmed": CMU(
+                category="pcr_tests_positive",
+                measurement="cumulative",
+                unit="unique_people",
+            ),
+        }
+        out = (
+            df.melt(id_vars=["county"], value_vars=crename.keys())
```

Comment:
Sorry, this changed from underneath you --  we renamed `county` to `location_name` to more closely match the database schema.

I'm already in the file, so I'll make the change myself!